package.json

```diff
@@ -35,7 +35,7 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "0.802.1",
     "@angular/animations": "8.2.1",
-    "@angular/cli": "8.2.1",
+    "@angular/cli": "^8.2.2",
```

Comment:
revert this, we use pinned dependencies since we use the Renovate bot ( https://github.com/azerothcore/Keira3/pull/14 )